shared/actions/fs/index.js

```diff
@@ -47,6 +47,7 @@ const makeEntry = (d: RPCTypes.Dirent, children?: Set<string>) => {
       return Constants.makeFolder({
         ...direntToMetadata(d),
         children: I.Set(children),
+        progress: children ? 'loaded' : undefined,
```

Comment:
This is the fix. The container uses `progress` field to figure out whether a folder is really empty or just not loaded.